### PR TITLE
Add public sort logic to List class.

### PR DESCRIPTION
**Pull request by jstevenco**
Default invocation will sort alpha (item text) in ascending order.

### CODE REVIEW

File: gkeepapi/node.py [hunk 1]
```diff
@@ -1420,6 +1421,16 @@ def _items(self, checked=None):
             )
         ])
 
+    def sort(self, key=attrgetter('text'), reverse=False):
+        sorted_children = sorted([node for node in self.children
```
Comment (kiwiz) [inline gkeepapi/node.py:1425]: We can just use `_items()` here.

File: gkeepapi/node.py [hunk 2]
```diff
@@ -1420,6 +1421,16 @@ def _items(self, checked=None):
...
+                                  if isinstance(node, ListItem) and not node.deleted],
+                                 key=key, reverse=reverse)
+        sort_value = random.randint(1000000000, 9999999999)
+        for node in sorted_children:
+            target_node = self.get(node.id)
```
Comment (kiwiz) [inline gkeepapi/node.py:1430]: Any reason why node isn't directly accessed here?

File: gkeepapi/node.py (hunk 1 above)
Comment (jstevenco) [reply gkeepapi/node.py]: Done.

File: gkeepapi/node.py (hunk 2 above)
Comment (jstevenco) [reply gkeepapi/node.py]: Just a brain fart on my part really -- not thinking that the ListItems themselves were the same objects -- fixed.

**Comment by kiwiz** [comment]
Merged, thx!

**Pull request merged**